db/migrate/20200625135809_update_states.rb

```diff
@@ -0,0 +1,9 @@
+class UpdateStates < ActiveRecord::Migration[6.0]
+  def up
+    MigrationHelpers::StateChanger.new(dummy_run: true).up
```

Comment:
Shouldn't this be `dummy_run: false` to actually apply the changes?

Reply:
YES!